src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/repositories/RemoteControllerRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.repositories;
+
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.RemoteController;
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public interface RemoteControllerRepository extends JpaRepository<RemoteController, Long> {
```

Comment:
Het is misschien een beetje gekke comment  maar je hebt de remote controller entity controller genoemd. Wat een beetje dubieus is aangezien er ook met controllers werken. Het is technisch niet fout maar kan een beetje verwarrend over komen.